packages/arb-token-bridge-ui/src/components/TransferPanel/ClaimableCardConfirmed.tsx

```diff
@@ -50,9 +50,25 @@ export function ClaimableCardConfirmed({ tx }: { tx: MergedTransaction }) {
 
   const networkName = getNetworkName(toNetworkId)
 
-  const currentChainIsValid =
-    (isSourceChainIdEthereum && isArbitrum) ||
-    (isSourceChainIdArbitrum && isEthereum)
+  const currentChainIsValid = useMemo(() => {
```

Comment:
@chrstph-dvx pls sanity check this as it touches CCTP

Reply:
I think that would break CCTP claim button. If it's a CCTP transfer, and L2 is an orbit chain, we should have claim button disabled. I think, we could check `!tx.isCctp && isWithdrawalSourceOrbitChain` instead

Reply:
Ok, I was thinking that CCTP wouldn't work on Orbit for some time but it's good to cover for it.